ansible/roles/disable_hyperthreading/README.md

```diff
@@ -0,0 +1,40 @@
+Disable Hyper-Threading
+=========
+
+Disable hyper-threading at runtime by systemd script at startup.
+
+See https://access.redhat.com/solutions/352663
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+dh_active - if true, hyper-threading is disabled
```

Comment:
It might just be me but I would find the reversed logic to be more straightforward ?
By that, I mean the role would be "hyperthreading" which would default to be enabled (true) and could be disabled (by setting false)

Reply:
That is the default unless disabled by something else like nosmt boot param.  Service and script will not be deployed unless you want to disable HT.  Alternately, they could be deployed, just not used.  Though I don't know what the value of that would be other than consistency across all node types, other than which (compute) have it enabled.